rules/falco_rules.yaml

```diff
@@ -1039,9 +1039,9 @@
                proc.cmdline startswith "agent.py /opt/datadog-agent")
               and fd.name startswith "/etc/dd-agent")
 
+# TODO: have a better way to identify the image repo (public vs internal)
 - macro: rancher_writing_conf
-  condition: (container.image.repository in (rancher_images)
-              and proc.name in (lb-controller,rancher-dns,healthcheck,rancher-metadat)
+  condition: (proc.name in (lb-controller,rancher-dns,healthcheck,rancher-metadat)
```

Comment:
I think the reason I qualified the macro with the image is that the process name 'healthcheck' was really generic. Maybe we could continue to have the image check just for that process?

Reply:
I agree `healthcheck` is too generic. How about removing `healthcheck` for now.

Reply:
added issue: https://github.com/falcosecurity/falco/issues/649

Reply:
Thanks! I think you can close the issue now given your later commits.